### DISCUSSION TITLE
here's the rest of my drawings!

### DISCUSSION THREAD

**Discussion by utkuasd** [stellar/sorobanathon#124] [General]
hey! i just finished drawing all of the stories! all chapters were really exciting... thanks for that!
 here is the old post: [https://github.com/stellar/sorobanathon/discussions/100#discussion-4622975](url)
--
chapter 4 (i accidentally wrote chapter 3 in the description) : https://litemint.com/items/GATSMF43BI6VIA2RMWKZCX2TWKHYW5VXVNC66EAJXLVQVIPDNDIXFTF5/NFT

chapter 5: https://litemint.com/items/GD3HFMZS3OGZUBHE2FDUMRKQDXITHDR7KPILOZRNN7MH5IT6GFIZCRWL/NFT

chapter 6: https://litemint.com/items/GBA7EM42VYQHCKEVCZXK6PYMJECCZO5EPO7TL3UET3CA5KTHPSND7NPZ/NFT

**Comment by stellarjohn** [comment]
Hi utkuasd!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 3900 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
